### PR TITLE
FWI-1093 - Trivy 0.12.0 does not bring log4j vulns - update to 0.21.3

### PR DESCRIPTION
**Pull request by jdesouza**
## Checklist

* [ ] I have updated the CHANGELOG of any plugins updated
* [ ] I have updated the version file of any plugins updated
* [ ] I have added tests wherever appropriate
* [ ] I have labeled this PR with a label per plugin updated

[YouTrack FWI-1093](https://fairwinds.myjetbrains.com/youtrack/issue/FWI-1093)

### CODE REVIEW

**Review by rbren — COMMENTED** [review]
Thanks James! Let's make sure Barnie agrees with our thinking here tomorrow but I think this will do the trick!

File: plugins/trivy/pkg/image/scan.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func ScanImageFile(imagePath string, imageID string, tempDir string) ([]models.V
 		os.Remove(reportFile)
 	}()
 
-	report := []models.VulnerabilityList{}
+	report := models.TrivyResults{}
```
Comment (makoscafee) [inline plugins/trivy/pkg/image/scan.go:118]: mmh why did you need to wrap this to  new struct?

File: plugins/trivy/pkg/models/image.go [hunk 2]
```diff
@@ -30,6 +30,10 @@ type ImageReport struct {
 	Report         []VulnerabilityList
 }
 
+type TrivyResults struct {
```
Comment (makoscafee) [inline plugins/trivy/pkg/models/image.go:33]: why do we need this?

Comment (jdesouza) [reply plugins/trivy/pkg/models/image.go:33]: We update trivy version because old version is not report with all vulns, but trivy report updated their json schema and added a new Results tag.

File: plugins/trivy/pkg/image/scan.go (hunk 1 above)
Comment (jdesouza) [reply plugins/trivy/pkg/image/scan.go:118]: Added a Results tag for new trivy report schema version

**Review by makoscafee — APPROVED** [review]
